zerver/openapi/javascript_examples.js

```diff
@@ -252,7 +252,7 @@ add_example("set_typing_status", "/typing:post", 200, async (client, console) =>
         to: [user_id1, user_id2],
     };
 
-    // The user has started to type in the group PM with Iago and Polonius
+    // The user has started to type in the group DM with Iago and Polonius
```

Comment:
This is a nit, but we've generally been trying to just spell these out, say "group direct message", in documentation, since we generally avoid acronyms as a project.

Reply:
Done in the updated version of these changes. Added a line break so that it renders clearly in the documentation.